### ISSUE TITLE
Unable to read through all bar charts if sim is playing [VO]

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/membrane-transport#433]
**Test device**
MacBook Air (M1 chip)

**Operating System**
15.6

**Browser**
Safari 18.6

**Problem description**
For https://github.com/phetsims/qa/issues/1290, with VoiceOver I was unable to move through all of the bar charts to hear their descriptions. I can see them all listed in the a11yview and I can hear all of them when the sim is paused. I'm not sure if this is happening because VO keeps trying to interrupt with context responses or not. 

This seems to be limited to VO. I tried it with JAWS and @jessegreenberg tried it with NVDA and there wasn't a problem. 

**Steps to reproduce**
1. Turn on VO
2. On the Simple Diffusion screen, add some CO2 to either side 
3. Add some Potassium ions to either side
4. Tab to the Solute Concentrations Accordion box
5. Use VO keys + right arrow to try and move through all of the bar chart descriptions

**Visuals**

This is a small snippet, otherwise the file size was too big:

https://github.com/user-attachments/assets/976095f3-eecf-4de7-9611-3711f87ab2b5

**Comment by Nancy-Salpepi** [comment]
Every time the word "bullet" is read that was when I pressed the arrow keys. 
The two-tone beep isn't heard when it starts back from the beginning with oxygen.

**Comment by samreid** [comment]
@jessegreenberg and I wrote more about the general root problem in https://github.com/phetsims/scenery/issues/1731 and will proceed with a sim-specific case-specific workaround until we know more.

**Comment by Nancy-Salpepi** [comment]
Working great now @samreid and @jessegreenberg !!
Since you opened https://github.com/phetsims/scenery/issues/1731, is it OK to close this one?

**Comment by jessegreenberg** [comment]
Great, thanks @Nancy-Salpepi. Yes, ok to close this one we will continue in scenery.
